lib/cocoapods/installer/xcode/pods_project_generator.rb

```diff
@@ -211,7 +211,7 @@ def wire_target_dependencies(project, target_installation_results)
             # First, wire up all resource bundles.
             pod_target_installation_result.resource_bundle_targets.each do |resource_bundle_target|
               native_target.add_dependency(resource_bundle_target)
-              if pod_target.requires_frameworks? && pod_target.should_build?
+              if pod_target.build_as_dynamic_framework? && pod_target.should_build?
```

Comment:
I think this is not the same as it used to be before

Reply:
I confirmed that the[ static_framework pod](https://github.com/firebase/firebase-ios-sdk/blob/master/FirebaseInAppMessagingDisplay.podspec) with resources still works.